react/features/filmstrip/components/Filmstrip.web.js

```diff
@@ -122,9 +129,8 @@ class Filmstrip extends Component<*> {
          * will get updated without replacing the DOM. If the known DOM gets
          * modified, then the views will get blown away.
          */
-
-        const filmstripClassNames = `filmstrip ${_remoteVideosVisible ? ''
-            : 'hide-videos'}`;
+        const filmstripClassNames = `filmstrip ${_remoteVideosVisible
+            ? '' : 'hide-videos'} ${_toolboxVisible ? 'reduce-height' : ''}`;
```

Comment:
For the horizontal toolbar, the filmstrip needs to shrink in height when the toolbar is visible. The added CSS class should do nothing in the case of the old toolbar because the class should be scoped to the new-toolbox class on the body.